### PR TITLE
Add Resources Deployed Page

### PR DESCRIPTION
**Pull request by rchristman89**

### CODE REVIEW

File: articles/DeployedResources.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Resources Deployed
+
+The following resources are deployed during the Azure Virtual Desktop Accelerator. Some resources do not deploy and are dependent on a condition.
+
+|Resource |Instances |Condition |
+--- | --- | --- |
+Virtual Machine | Number of VMs Deployed | |
+Network Interface |  Number of VMs Deployed | |
+Disk (OS Disk) |  Number of VMs Deployed | |
+Virtual Network | 1 | If deployment is not On-Prem and an existing one is used |
```
Comment (regedit32) [inline articles/DeployedResources.md:10]:
```suggestion
Virtual Network | 1 | If Azure Active Directory domain-join is selected |
```

File: articles/DeployedResources.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Network Security Group | 1 | If deployment is not On-Prem and an existing one is used |
+Host Pool | 1 | |
+Workspace | 1 | |
+Application Group | 1 | |
+Key Vault | 1 | User Accounts are created during deployment |
```
Comment (regedit32) [inline articles/DeployedResources.md:15]: To keep it the same as the related condition
```suggestion
Key Vault | 1 | When Azure AD Accounts are created |
```

File: articles/DeployedResources.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+Custom Script Extension | 1 | STIGs are applied |
```
Comment (regedit32) [inline articles/DeployedResources.md:16]:
```suggestion
Custom Script Extension | 1 | When STIGs are applied |
```

File: articles/DeployedResources.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+Microsoft Powershell DSC | 1 | STIGs are applied |
```
Comment (regedit32) [inline articles/DeployedResources.md:17]:
```suggestion
Microsoft Powershell DSC | 1 | When STIGs are applied |
```

File: articles/DeployedResources.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+VM Sizing Deployment Script | 1 | |
+Directory Management Deployment Script | 1 | |
+Account Creation Deployment Script | 1 | Azure AD Accounts are created |
```
Comment (regedit32) [inline articles/DeployedResources.md:20]:
```suggestion
Account Creation Deployment Script | 1 | When Azure AD Accounts are created |
```

File: articles/DeployedResources.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+Disk (OS Disk) |  Number of VMs Deployed | |
+Virtual Network | 1 | If deployment is not On-Prem and an existing one is used |
+Network Security Group | 1 | If deployment is not On-Prem and an existing one is used |
+Host Pool | 1 | |
+Workspace | 1 | |
+Application Group | 1 | |
+Key Vault | 1 | User Accounts are created during deployment |
+Custom Script Extension | 1 | STIGs are applied |
+Microsoft Powershell DSC | 1 | STIGs are applied |
+VM Sizing Deployment Script | 1 | |
```
Comment (regedit32) [inline articles/DeployedResources.md:18]:
```suggestion
VM Sizing Deployment Script | 1 | When Host Pool type is _pooled_ |
```

File: articles/DeployedResources.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+The following resources are deployed during the Azure Virtual Desktop Accelerator. Some resources do not deploy and are dependent on a condition.
+
+|Resource |Instances |Condition |
+--- | --- | --- |
+Virtual Machine | Number of VMs Deployed | |
+Network Interface |  Number of VMs Deployed | |
+Disk (OS Disk) |  Number of VMs Deployed | |
+Virtual Network | 1 | If deployment is not On-Prem and an existing one is used |
+Network Security Group | 1 | If deployment is not On-Prem and an existing one is used |
```
Comment (regedit32) [inline articles/DeployedResources.md:11]:
```suggestion
Network Security Group | 1 | If Azure Active Directory domain-join is selected |
```

**Review by regedit32 — CHANGES_REQUESTED** [review]

File: articles/DeployedResources.md (hunk 7 above)
Comment (rchristman89) [reply articles/DeployedResources.md:11]: This happens in 2 scenarios, On-Prem if the choose to also create on.

Comment (rchristman89) [reply articles/DeployedResources.md]: Fixed

**Review by regedit32 — APPROVED** [review]

**Pull request merged**